crates/libs/result/Cargo.toml

```diff
@@ -13,6 +13,8 @@ categories = ["os::windows-apis"]
 [features]
 default = ["std"]
 std = []
+# slim-errors reduces the size of Error (and Result<()>) to that of a single HRESULT
+slim-errors = []
```

Comment:
Cargo features are meant to be additive, so I think this should be in the affirmative - e.g. `rich-errors` - and enabled by default. You can then use `no-default-features` to remove it just like you would for the `std` feature.

Reply:
I took that approach to begin with, but it's quite error prone.  You have to make sure that every dependency on `windows-result` uses `no-default-features`, including the dependency from `windows-core`.

If change the `windows-core` --> `windows-result` dependency to be `no-default-features`, then everyone will get slim errors by default, unless they specifically opt-in to `rich-errors`.  I went pretty far down that path, and I think it makes things worse than treating "slim errors" as a feature.

I'm open to changing this, if the result looks good.

Reply:
After a bunch of iterations, we converged on using a `--cfg` option to control this.  It's not perfect, but it works.